### PR TITLE
Security holoprojector resprite

### PR DESCRIPTION
**Pull request by Kit0vras**
## About the PR

**Media**

https://user-images.githubusercontent.com/123590995/234216214-53e8218a-a3fe-4b51-acaa-06aff8b17536.mp4

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: The security holoprojector sprite has been changed.

### CODE REVIEW

File: Resources/Textures/Structures/Holo/security.rsi/meta.json [hunk 1]
```diff
@@ -5,12 +5,16 @@
     "y": 32
   },
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/f4017da82ae4de7bfaf8ebdbba33d0de81c15587",
+    "copyright": "Created by Kit0vras#5869",
```
Comment (AJCM-git) [inline Resources/Textures/Structures/Holo/security.rsi/meta.json:8]:
```suggestion
    "copyright": "Made by Kit0vras#5869 (discord) / Kit0vras (github) for SS14",
```

File: Resources/Textures/Structures/Holo/security.rsi/meta.json [hunk 2]
```diff
@@ -5,12 +5,16 @@
...
             "name": "icon",
             "delays": [
                 [
+                    0.3,
+                    0.3,
+                    0.3,
+                    0.3,
                     0.3,
                     0.3,
                     0.3,
```
Comment (AJCM-git) [inline Resources/Textures/Structures/Holo/security.rsi/meta.json:20]: Make the delays a lil shorter so the animation goes quicker, right now it looks laggy

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Textures/Structures/Holo/security.rsi/meta.json (hunk 2 above)
Comment (Kit0vras) [reply Resources/Textures/Structures/Holo/security.rsi/meta.json:20]: I will add more frames as shorter delays didn't solve the problem.

**Comment by Kit0vras** [comment]
I made more frames and reduced the delay. Is that better?

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**